src/service/url-replacements-impl.js

```diff
@@ -76,7 +88,7 @@ export class UrlReplacements {
     this.initialized_ = true;
     // Returns a random value for cache busters.
     this.set_('RANDOM', () => {
-      return Math.random();
+      return String(Math.random());
```

Comment:
I wanted to protect against returning promises for the sync case.  I could extend the typedef to be number|string|bool|undefined.  In the end, these will be written into the URL so they should be values that make sense to do so (e.g. not object, function, or promise as mentioned).  Thoughts?